### PR TITLE
Release/5.17.0

### PR DESCRIPTION
**Pull request by jaybeckr**
Fixes # .

#### Additional details about this PR

### CODE REVIEW

**Comment by dimitriBouteille** [comment]
Good news ❤️

**Review by akshitaWaldia — APPROVED** [review]

**Review by prayagr — APPROVED** [review]

File: Controller/Checkout/CompleteSession.php [hunk 1]
```diff
@@ -58,7 +59,12 @@ class CompleteSession extends \Magento\Framework\App\Action\Action
     private $cookieMetadataFactory;
 
     /**
-     * CompleteCheckout constructor.
+     * @var \Magento\Checkout\Model\Session
+     */
+    protected $magentoCheckoutSession;
```
Comment (jajajaime) [inline Controller/Checkout/CompleteSession.php:64]: This property and its dependency injection are not used. We should remove it.

File: Model/Resolver/CheckoutSessionDetails.php [hunk 2]
```diff
@@ -42,6 +42,7 @@ public function __construct(
      * @return array
      * @throws GraphQlInputException
      */
+
```
Comment (jajajaime) [inline Model/Resolver/CheckoutSessionDetails.php:45]: Extra line not needed.

File: Controller/Checkout/CompleteSession.php (hunk 1 above)
Comment (sgabhart22) [reply Controller/Checkout/CompleteSession.php]: Correct you are, this (along with another unused `$amazonCheckoutSession`) has been removed from this class.

File: Model/Resolver/CheckoutSessionDetails.php (hunk 2 above)
Comment (sgabhart22) [reply Model/Resolver/CheckoutSessionDetails.php]: Extra line removed!

**Pull request merged**